pkg/cmd/admin/registry/registry.go

```diff
@@ -74,7 +75,14 @@ type RegistryConfig struct {
 
 var errExit = fmt.Errorf("exit")
 
-const defaultLabel = "docker-registry=default"
+const (
+	defaultLabel = "docker-registry=default"
+	defaultPort  = 5000
+	// TODO: `/healthz` has been deprecated by `/`; keep it temporarily for
+	// backwards compatibility
+	healthzRoute               = "/healthz"
```

Comment:
*How* is simple. We'll just replace `/healthz` with `/` here. Our registry supports both endpoints for some time now.

*When* is more interesting. I guess the next major release would be a good candidate. Or the release with a strict requirement on newer registry version due to api changes. Note this patch doesn't introduce the `/healtz` endpoint, it just moves its definition here from its [original place](https://github.com/openshift/origin/pull/6439/files#diff-df3b6baf7e3b18ed4b1ff84012467504L260).

Shall I make the comment more specific about the *when*?